mpas_analysis/ocean/climatology_map_argo.py

```diff
@@ -0,0 +1,461 @@
+            raise ValueError('config section {} does not contain valid '
+                             'list of depths'.format(sectionName))
+
+        # the variable 'timeMonthly_avg_landIceFreshwaterFlux' will be added to
+        # mpasClimatologyTask along with the seasons.
+        remapClimatologySubtask = RemapDepthSlicesSubtask(
+            mpasClimatologyTask=mpasClimatologyTask,
+            parentTask=self,
+            climatologyName=fieldName,
+            variableList=[mpasFieldName],
+            seasons=seasons,
+            depths=depths,
+            comparisonGridNames=comparisonGridNames,
+            iselValues=iselValues)
+
+        if refConfig is None:
+
+            refTitleLabel = 'Roemmich-Gilson Argo Climatology: Salinity'
+
+            observationsDirectory = build_config_full_path(
+                config, 'oceanObservations', 'argoSubdirectory')
+
+            obsFileName = \
+                '{}/ArgoClimatology_TS.nc'.format(
+                        observationsDirectory)
+            refFieldName = 'salinity'
+            outFileLabel = 'salinArgo'
+            galleryName = 'Roemmich-Gilson Climatology: Argo'
+            diffTitleLabel = 'Model - Argo'
+
+            remapObservationsSubtask = RemapArgoClimatology(
+                    parentTask=self, seasons=seasons, fileName=obsFileName,
+                    outFilePrefix=refFieldName,
+                    fieldName=refFieldName,
+                    depths=depths,
+                    comparisonGridNames=comparisonGridNames)
+
+            self.add_subtask(remapObservationsSubtask)
+
+        else:
+            remapObservationsSubtask = None
+            refRunName = refConfig.get('runs', 'mainRunName')
+            galleryName = None
+            refTitleLabel = 'Ref: {}'.format(refRunName)
+
+            refFieldName = mpasFieldName
+            outFileLabel = 'salin'
+            diffTitleLabel = 'Main - Reference'
+
+        for comparisonGridName in comparisonGridNames:
+            for season in seasons:
+                for depth in depths:
+                    subtask = PlotClimatologyMapSubtask(
+                        parentTask=self,
+                        season=season,
+                        comparisonGridName=comparisonGridName,
+                        remapMpasClimatologySubtask=remapClimatologySubtask,
+                        remapObsClimatologySubtask=remapObservationsSubtask,
+                        refConfig=refConfig,
+                        depth=depth)
+
+                    subtask.set_plot_info(
+                        outFileLabel=outFileLabel,
+                        fieldNameInTitle='Salinity',
+                        mpasFieldName=mpasFieldName,
+                        refFieldName=refFieldName,
+                        refTitleLabel=refTitleLabel,
+                        diffTitleLabel=diffTitleLabel,
+                        unitsLabel=r'PSU',
+                        imageCaption='ARGO_Salinity',
```

Comment:
Similarly to above, "Salinity compared with ARGO observations"

Reply:
In both cases, you could also put the depth in the caption if you wanted but it's not necessary.

Reply:
done